src/main/java/uk/ac/ebi/eva/bd2k/export/StudyExporter.java

```diff
@@ -59,6 +61,9 @@ public void export(List<T> studies, Path outputDirectory) {
             logger.info("Done");
         } catch (FileNotFoundException e) {
             logger.error("Cannot create output file: {}", e.getMessage());
+        } catch (ProjectFirstPublicDateNotFoundException e) {
+            logger.error(e.getMessage());
+            System.exit(1);
```

Comment:
Applications shouldn't be exited from inside the business logic, but the main method should be the one responsible for capturing breaking exceptions and exiting.

I would say that the `FileNotFoundException` should be treated the same way.